app/assets/javascripts/controllers/boardController.js

```diff
@@ -0,0 +1,24 @@
+app.controller('BoardCtrl', ['$scope', '$location', 'Restangular', 'Session', 'Boards', function($scope, $location, Restangular, Session, Boards){
```

Comment:
Code looks great, very modular. Controllers limited to what they're intended for. Comments would put your code over the top and make this an exceptional app